contribs/gmf/src/directives/importdatasource.js

```diff
@@ -167,6 +188,25 @@ gmf.ImportdatasourceController = class {
    */
   $onInit() {
     this.gmfExternalDataSourcesManager_.map = this.map;
+
+
+    if (this.serversEngine_) {
+      // Timeout to let Angular render the placeholder of the input properly,
+      // otherwise typeahead would copy the string with {{}} in it...
+      this.timeout_(() => {
+        const engine = goog.asserts.assert(this.serversEngine_);
```

Comment:
I find this a bit complicated and uneasy to read; what about simply have:
```
goog.asserts.assert(this.serversEngine_);
```
and use `this.serversEngine_.ttAdapter()` bellow